modules/enabling-scanner-v4-after-helm-installation.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-after-helm-installation_{context}"]
+= Enabling RHACS Scanner V4 after Helm installation
+
+[role="_abstract"]
+Scanner V4 is not enabled by default, but you can enable Scanner V4 after installation.
+
+If you want to enable {product-title-short} on secured clusters by using Helm charts, you must first enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. Run the command to update {product-title-short} with customizations as described in "Changing configuration options after deploying the central-services Helm chart":
++
+[source,terminal]
+----
+helm upgrade -n stackrox \
```

Comment:
Unless there is a reason to exclude it, this command needs a $ prompt at the beginning:
```suggestion
$ helm upgrade -n stackrox \
```